README.md

```diff
@@ -6,7 +6,6 @@ provides a curated and maintained set of build [tools](./tools) for Go projects.
 [![Release](https://github.com/einride/sage/actions/workflows/release.yml/badge.svg)](https://github.com/einride/sage/actions/workflows/release.yml)
 
 ## Requirements
-
```

Comment:
This incorrect formatting will be fixed by the GHA job, leading to a dirty workspace, but the job will still be OK